public/style.css

```diff
@@ -0,0 +1,6 @@
+body {
+  background-color: #e5e5e5;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5830965)